### ISSUE TITLE
Cockpit next?

### ISSUE THREAD

**Issue by gryphonmyers** [piotr-cz/cockpit-import#6]
any plans to update your add-on for the new version?

**Comment by piotr-cz** [comment]
Due to my current workload, not any time soon.

The thing is that current module version uses heavily angular which has been dropped in cockpit-next in favor of Riot.js which I have no experience with.

As cockpit requires modern browser to run, some angular utilities could be moved native (Promise, XHR2)

I'll keep this issue open until I'll be able to tell more.

**Comment by aheinze** [comment]
@piotr-cz I started to work on an import functionality in cockpit next, feel invited to join ;-)

**Comment by piotr-cz** [comment]
I'd feel more comfortable if Import functionality would be a separate and independent module.
It's quite complex and I think it deserves own repository, decoupled from Cockpit repo

**Comment by gryphonmyers** [comment]
I disagree - I think the core integration makes perfect sense how Artur
laid it out. Collection export has always been a core feature, and the lack
of an accompanying import made it seem incomplete. Putting the functions
right there in the context of the collection you're looking at seems like
correct ux.

**Comment by piotr-cz** [comment]
In my opinion it makes sense to have built-in import functionality for JSON files previously exported from Cockpit-next or Cockpit where data structure is very similar.

But for me personally main goal for this module is to import CSV/ spreadsheet data, that I receive from customer, with for example gelocating addresses.
